src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -122,10 +129,10 @@ private function listMessages(SymfonyStyle $io, int $max)
         $io->comment('Run <comment>messenger:failed:show {id} -vv</comment> to see message details.');
```

Comment:
this line (and a few other ones in this command... and maybe other commands, like`:retry` and `:remove`) now needs to include `--failure-transport=` if that option was passed. Otherwise, you will be looking for the message in the wrong transport.

Btw, with Doctrine as your failure transport, forgetting the `--failure-transport=` option actually DOES work... but due to a bug I just discovered (that we should fix, in a different PR). The `find()` method in the Doctrine connection class incorrectly does NOT include the `AND queue_name = ?` part. https://github.com/symfony/symfony/blob/2da27bbd2751f51480e49f6b9132fa45e2a14f9e/src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php#L278-L281